src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/config/CacheConfiguration.kt

```diff
@@ -26,7 +26,7 @@ class CacheConfiguration(private val buildProperties: BuildProperties) {
   @Bean
   fun cacheManager(
     connectionFactory: RedisConnectionFactory,
-    @Value("\${spring.cache.type}") isCacheEnabled: String = "none",
+    @Value("\${spring.cache.type:none}") isCacheEnabled: String,
   ): CacheManager {
     if (isCacheEnabled.equals("none", ignoreCase = true)) {
```

Comment:
Sorry, didn't think of this in the previous review: could you have the `else` case be an `else if` checking whether the cache type is set to `redis`, and have an `else` case that fails with something along the lines of `Unsupported cache type: <the-read-value>`, please? Just to make sure nobody simply changes `spring.cache.type` to some other caching library and thinks that's all they need to do to change the cache.